gh-sprints/statistics.py

```diff
@@ -0,0 +1,41 @@
+from operator import attrgetter
+
+
+class LabelStatistics(object):
+
+    def __init__(self, name, collection):
+        self.name = name
+        self.issues = []
+        self.total_story_points = 0
+        self.collection = collection
+
+    def add_issue(self, issue):
+        self.issues.append(issue)
+        self.total_story_points += issue.points
+
+    @property
+    def story_points_as_percentage(self):
+        percentage = (float(self.total_story_points) / float(self.collection.completed_story_points)) * 100
```

Comment:
if you do `from __future__ import division`, you get floating point division by default and can leave it out here.